rmc/utils/mpc.py

```diff
@@ -173,139 +167,21 @@ def import_grantham():
-    :return: None; function writes Table to resource path.
-    """
-    logger.info("Importing set of transcripts to keep...")
-    transcripts = get_constraint_transcripts(outlier=False)
-
-    logger.info(
-        "Reading in VEP context HT and filtering to missense variants in canonical transcripts..."
-    )
-    # Using vep_context.path to read in table with fewer partitions
-    # VEP context resource has 62164 partitions
-    ht = (
-        hl.read_table(vep_context.path, _n_partitions=n_partitions)
-        .select_globals()
-        .select("vep", "was_split")
-    )
-    ht = process_vep(ht, filter_csq=True, csq=missense_str)
-    ht = ht.filter(transcripts.contains(ht.transcript_consequences.transcript_id))
-    ht = get_annotations_from_context_ht_vep(ht)
-    # Save context Table to temporary path with specified deletion policy because this is a very large file
-    # and relevant information will be saved at `context_with_oe` (written below)
-    # Checkpointing here to force this expensive computation to compute
-    # before joining with RMC tables in `get_oe_annotation`
-    # This computation is expensive, so overwrite only if specified (otherwise, read existing file)
-    ht = ht.checkpoint(
-        f"{TEMP_PATH_WITH_SLOW_DEL}/vep_context_mis_only_annot.ht",
-        _read_if_exists=not overwrite_temp,
-        overwrite=overwrite_temp,
-    )
-
-    logger.info(
-        "Adding regional missense constraint missense o/e annotation and writing to resource path..."
-    )
-    ht = get_oe_annotation(ht)
-    ht = ht.key_by("locus", "alleles", "transcript")
-    ht = ht.checkpoint(
-        context_with_oe.path,
-        overwrite=overwrite_output,
-        _read_if_exists=not overwrite_output,
-    )
-    logger.info("Output OE-annotated context HT fields: %s", set(ht.row))
-
-    logger.info(
-        "Creating dedup context with oe (with oe and transcript annotations only)..."
-    )
-    ht = ht.select("oe").key_by("locus", "alleles")
-    ht = ht.collect_by_key()
-    ht = ht.annotate(**{"oe": hl.nanmin(ht.values.oe)})
-    ht = ht.annotate(
-        **{"transcript": ht.values.find(lambda x: x.oe == ht.oe).transcript}
-    )
-    ht = ht.drop("values")
-    ht.write(
-        context_with_oe_dedup.path,
-        overwrite=overwrite_output,
-    )
-    logger.info("Output OE-annotated dedup context HT fields: %s", set(ht.row))
-
-
 def prepare_pop_path_ht(
     gnomad_data_type: str = "exomes",
     af_threshold: float = 0.001,
     overwrite_temp: bool = False,
     overwrite_output: bool = True,
+    freeze: int = CURRENT_FREEZE,
 ) -> None:
     """
     Prepare Table with 'population' (common gnomAD missense) and 'pathogenic' (ClinVar pathogenic/likely pathogenic missense) variants.
 
     .. note::
-        This function reads in data from a requester-pays bucket and will fail if requester-pays
+        - This function reads in data from a requester-pays bucket and will fail if requester-pays
```

Comment:
ooh I missed this last time, but the entire RMC bucket is requester-pays now, so we should probably remove this
```suggestion
```